lib/snapshot/snapshot_config.rb

```diff
@@ -140,6 +143,27 @@ def project_name
       File.basename(self.project_path, ".*" )
     end
 
+    # Returns the file name of the build product
+    def app_name
+      result = manual_app_name;
+
+      unless result
+        project_key = 'project'
+        project_key = 'workspace' if project_path.end_with?'.xcworkspace'
+        command = "xcodebuild -#{project_key} '#{project_path}' -showBuildSettings | grep 'WRAPPER_NAME' | sed 's/[ ]*WRAPPER_NAME = //'"
+        Helper.log.debug command
+        
+        result = `#{command}`.strip!
+        Helper.log.debug "Found app name: #{result}"
+      end
+
+      unless result
```

Comment:
How about changing that to just `return nil` and add a `||` when it's used, see other comment.